app/src/main/java/co/kr/mvvm_study/view/MainActivity.kt

```diff
@@ -6,16 +6,19 @@ import android.os.Bundle
 import android.widget.EditText
 import androidx.appcompat.app.AlertDialog
 import androidx.appcompat.app.AppCompatActivity
+import androidx.lifecycle.ViewModelProvider
 import androidx.recyclerview.widget.LinearLayoutManager
 import co.kr.mvvm_study.R
 import co.kr.mvvm_study.databinding.ActivityMainBinding
 import co.kr.mvvm_study.model.ToDoModel
 import co.kr.mvvm_study.view.adapter.ToDoListAdapter
+import co.kr.mvvm_study.viewmodel.ToDoViewModel
 import java.util.*
 import kotlin.collections.ArrayList
 
 class MainActivity : AppCompatActivity() {
     private lateinit var binding: ActivityMainBinding
+    private lateinit var toDoViewModel: ToDoViewModel
```

Comment:
```suggestion
       private val toDoViewModel : ToDoViewModel by viewModels()
```

Android KTX를 사용하면 뷰모델 위임 초기화로 더 깔끔하게 생성할 수 있습니다~ 참고바람. 위임 초기화가 어떻게 동작하는 지도 알아보고 사용하면 더 좋을듯합니당 ^_^